x/fees/keeper/fee_decorator_test.go

```diff
@@ -0,0 +1,89 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/legacy/legacytx"
+
+	"github.com/desmos-labs/desmos/v3/x/fees/keeper"
+	"github.com/desmos-labs/desmos/v3/x/fees/types"
+	profilestypes "github.com/desmos-labs/desmos/v3/x/profiles/types"
+)
+
+func (suite *KeeperTestSuite) TestFeeDecorator() {
```

Comment:
Can you add a test which is when BlockHeight == 0?

Reply:
@dadamu added.

Reply:
I can't re-request you the review. I guess it's a GitHub bug.

Reply:
I can't  review as well. lol

Reply:
Never mind it is back.